pwa/src/services/stringToCssModuleSelector.tsx

```diff
@@ -0,0 +1,11 @@
+export const stringToCssModuleSelector = (string: any): string => {
+  function capitalize(str: any) {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+  }
+  function unCapitalize(str: any) {
+    return str.charAt(0).toLowerCase() + str.slice(1);
+  }
+
+  const newString = string.split(" ").map(capitalize).join(" ").replace(/\s/g, "").split(" ").map(unCapitalize);
+  return newString;
+};
```

Comment:
This seems quite complex. We have `lodash` included in our project, and I think [this function](https://www.geeksforgeeks.org/lodash-_-camelcase-method/) within it does the same.